src/populate/populate_block.cpp

```diff
@@ -42,13 +42,16 @@ populate_block::populate_block(dispatcher& dispatch, const fast_chain& chain)
 void populate_block::populate(block_const_ptr block,
     result_handler&& handler) const
 {
+    auto& header_ptr = block->header();
```

Comment:
Style: this is not a pointer, should be:

```cpp
const auto& header = block->header();
```

Reply:
understood, thanks.